src/geometry/manifold/sheaf/basic.lean

```diff
@@ -0,0 +1,69 @@
+/-
```

Comment:
I think `sheaf` is the correct answer (we consistently use singular rather than plural as far as I can tell -- `manifold`, `group`), but unfortunately the port has already reached the `Topology/Sheaves` folder so it's not easy to change now.  I suggest making the manifold one `sheaf` for now and renaming the topology one after the port?